mario/tools/tabletransform.py

```diff
@@ -232,3 +232,48 @@ def SUT_to_IOT(instance, method):
     rename_index(matrices["baseline"])
 
     return matrices, indeces, units
+
+
+def ISARD_TO_CHENERY_MOSES(instance,scenario):
```

Comment:
would be nice if you can also add some comments as docstring here to describe the source of function (refer to a study or something?)

Reply:
added in both methods!